kmir/src/kmir/__main__.py

```diff
@@ -141,6 +142,40 @@ def _kmir_prune(opts: PruneOpts) -> None:
     proof.write_proof_data()
 
 
+def _kmir_section_edge(opts: SectionEdgeOpts) -> None:
+    # Proof dir is checked to be some in arg parsing for instructive erros
```

Comment:
Do I understand this correctly that `opts.proof_dir` is never `None` here?

```suggestion
    # Proof dir is checked to be some in arg parsing for instructive errors
    assert opts.proof_dir is not None
```